### DISCUSSION TITLE
[ FF7 ] Failed to launch

### DISCUSSION THREAD

**Discussion by JettaNatioN** [julianxhokaxhiu/FFNx#187] [General]
i keep getting the failed to load no matter what i do on 7th heaven and no clue where to find the crash log that happens

### Launching Final Fantasy VII ###
Checking FF7 is not running ...
Checking FF7 .exe exists at C:\Games\Final Fantasy VII\FF7.exe ...
Verifying installed game is compatible ...
Verifying game is not installed in a System/Protected folder ...
Verifying English game files exist ...
Verifying game is full/max install ...
Creating missing required directories ...
Verifying additional files for 'battle' and 'kernel' folders exist ...
Verifying all movie files exist ...
Verifying music files exist ...
Verifying latest game driver is installed ...
	Copying C:\Users\windl\Downloads\7th_Heaven_2_2_3_522_Portable\Resources\Game Driver\GameDriver.cfg -> C:\Games\Final Fantasy VII\FFNx.cfg ...
	FFNx.dll file is up to date.
Verifying game driver shaders folders exist ...
Verifying ff7 exe ...
Checking a profile is active ...
Checking mod compatibility requirements ...
Checking mod constraints for compatibility ...
Checking mod load order requirements ...
Looking for game disc ...
Found game disc at F:\ ...
Applying changed values to registry ...
Copying ff7input.cfg to FF7 path ...
	using control configuration file [Default] Steam KB+PlayStation (Stock).cfg ...
	Copying C:\Users\windl\Downloads\7th_Heaven_2_2_3_522_Portable\Resources\Controls\[Default] Steam KB+PlayStation (Stock).cfg -> C:\Games\Final Fantasy VII\ff7input.cfg ...
Creating Runtime Profile ...
	adding paths to monitor ...
Copying EasyHook.dll to FF7 path (if not found or older version detected) ...
	skipped copying EasyHook.dll ...
	skipped copying EasyHook32.dll ...
Checking if Reunion mod is installed ...
	found: False
Launching additional programs to run (if any) ...
Writing temporary runtime profile file to C:\Users\windl\AppData\Local\Temp\tmpAA65.tmp ...
Attempting to inject with EasyHook ...
Failed to inject after max amount of tries (14) ...
Unknown error in injected assembler code.

**Comment by eve-atum** [comment]
Why are you asking FFNx about a 7H problem?

**Comment by JettaNatioN** [comment]
I have no idea where I'm meant to go sorry. Its just confusing and a lot of things keep sending to there



Sent from my Samsung Galaxy smartphone.



-------- Original message --------

**Comment by eve-atum** [comment]
Not sure why things are sending you here... it would make a lot more sense to go to the 7H group on the Qhimm forums: http://forums.qhimm.com/index.php?board=37.0

Or join their Discord: https://discord.com/invite/fyE2Vk2

**Comment by JettaNatioN** [comment]
It mentions something about a crash log but I don't know where that be and now it keeps bringing up what posted.
I'll have a look at the forums and see what I need to do there



Sent from my Samsung Galaxy smartphone.



-------- Original message --------

**Comment by julianxhokaxhiu** [comment]
Thanks @eve-atum for having my back.

As said, please approach the 7h author on Qhimm ( Forum or Discord ). Cheers :)
